methods/psqr.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name": "psqr",
+    "status": "registered",
+    "verifiableDataRegistry": "Public Square",
+    "contactName": "Christian Gribneau",
+    "contactEmail": "protocol@vpsqr.com",
```

Comment:
Are you good with editors assuming that anyone with this email domain is acceptable to make updates to this registration entry?

Reply:
> Are you good with editors assuming that anyone with this email domain is acceptable to make updates to this registration entry?

The email address `protocol@vpsqr.com` owns the open source code, standards, and distributions. It is a role that distributes to myself and one other person in the company.